src/chttpd/src/chttpd_auth_cache.erl

```diff
@@ -155,15 +155,17 @@ spawn_changes(Since) ->
     Pid.
 
 listen_for_changes(Since) ->
-    ensure_auth_ddoc_exists(dbname(), <<"_design/_auth">>),
+    DbName = dbname(),
+    erlang:put(io_priority, {system, DbName}),
```

Comment:
This would be a cluster level db name and in other case we use the shard names. That should be ok, the ioq servers can  handle both, right?